README.md

```diff
@@ -57,6 +57,11 @@ $app->get('/get', function (ServerRequestInterface $request, ResponseInterface $
 You can do this also in asynchronous contexts and long running processes,
 since no super-globals nor I/O are involved.
 
+It is recommended that you use a key with lots of entropy, preferably
```

Comment:
May I suggest also fixing the strings used in the various examples (so that it doesn't look like a password)?